src/game/GameProcMain.cpp

```diff
@@ -4906,9 +5029,17 @@ void CGameProcMain::MsgRecv_UserState(DataPack * pDataPack, int & iOffset) {
     } else if (N3_SP_STATE_CHANGE_ACTION == eSP) // ũ�� ����
     {
         if (1 == iState) {
-            pBPC->AnimationAdd(ANI_GREETING0, true); // �λ�
+            pBPC->AnimationAdd(ANI_GREETING0, true); // ???
+        } else if (2 == iState) {
+            pBPC->AnimationAdd(ANI_GREETING1, true); // ????
+        } else if (3 == iState) {
+            pBPC->AnimationAdd(ANI_GREETING2, true); // ????
         } else if (11 == iState) {
-            pBPC->AnimationAdd(ANI_WAR_CRY1, true); // ����
+            pBPC->AnimationAdd(ANI_WAR_CRY0, true); // ????
```

Comment:
I realise it's not logical, but officially 11 = `ANI_WAR_CRY1`, 12 = `ANI_WAR_CRY0` and 13 = `ANI_WAR_CRY2`. I made the exact same mistake.

```cpp
      case STATE_CHANGE_ACTION:
        switch ( iState )
        {
          case 1u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_GREETING0, 1);
            break;
          case 2u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_GREETING1, 1);
            break;
          case 3u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_GREETING2, 1);
            break;
          case 11u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_WAR_CRY1, 1);
            break;
          case 12u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_WAR_CRY0, 1);
            break;
          case 13u:
            CPlayerBase::AnimationAdd((CPlayerBase *)pBPC, ANI_WAR_CRY2, 1);
            break;
        }
        return;
```